packages/lesswrong/components/users/UsersProfile.jsx

```diff
@@ -25,161 +37,177 @@ const styles = theme => ({
       marginRight: 4
     }
   },
+  actions: {
+    marginLeft: 20,
+  },
+  bio: {
+    margin: 0,
+    paddingBottom: 10,
+    paddingLeft: 20,
+  
+    "& p": {
+      fontSize: 16,
+      lineHeight: 26,
```

Comment:
This will cause the line height to be `26 * em` which will make it really really really large (see attached screenshot)

![Screen Shot 2019-03-26 at 11 50 42 AM](https://user-images.githubusercontent.com/9090789/55025091-701ef100-4fbd-11e9-8a43-0deaa8b84cf8.png)